### PR TITLE
feat: adapt rules for our org

### PR DESCRIPTION
**Pull request by ddhorstman**
I've updated the README and included an `eslintrc` matching the rules we discussed!

### CODE REVIEW

File: linters/.eslintrc [hunk 1]
```diff
@@ -1,5 +1,86 @@
-// Use this file as a starting point for your project's .eslintrc.
-// Copy this file, and add rule overrides as needed.
 {
-  "extends": "airbnb"
```
Comment (abrindam) [inline linters/.eslintrc:4]: Is the idea we're just going to list all the airbnb rules explicitly?

File: README.md [hunk 2]
```diff
@@ -568,17 +568,17 @@ Other Style Guides
 ## Strings
 
   <a name="strings--quotes"></a><a name="6.1"></a>
```
Comment (abrindam) [inline README.md:570]: Are we going to change the title of the document? It's still "AirBNB Style Guide" or something like that.

**Review by abrindam — APPROVED** [review]
Few questions but looks good to me.

File: README.md (hunk 2 above)
Comment (ddhorstman) [reply README.md:570]: Good point! I'll do that.

File: linters/.eslintrc (hunk 1 above)
Comment (ddhorstman) [reply linters/.eslintrc:4]: Wow, good catch! This is a mistake - I accidentally removed `airbnb` from the `extends`. I'll be curious what other things we catch when I fix it.

**Pull request merged**